chroot-bin/croutoncycle

```diff
@@ -289,7 +299,7 @@ if [ "${destdisp#:}" = "$destdisp" ]; then
         # Raise the right window after chvting, so that it can update
         if [ "$tty" != 'tty1' ]; then
             sudo -n chvt 1
-            sleep .1
```

Comment:
I suggest keeping the sleep in there when chvting, otherwise the brightness might get reverted again.